<?xml version="1.0"?>
<launch>

	<node name="robotino_motion_server_node" pkg="robotino_motion" type="robotino_motion_server_node" output="screen">
		<param name="min_linear_vel" value="0.05"/>
		<param name="max_linear_vel" value="0.1"/>
		<param name="min_angular_vel" value="0.1"/>
		<param name="max_angular_vel" value="0.3"/>
		<param name="angular_acc" value="0.2"/>
		<param name="percentage" value="0.1"/>
		<param name="linear_acc" value="0.2"/>
	</node>
</launch>
